files/file.go

```diff
@@ -271,7 +272,23 @@ func (i *FileInfo) detectSubtitles() {
 	i.Subtitles = []string{}
 	ext := filepath.Ext(i.Path)
 
-	// TODO: detect multiple languages. Base.Lang.vtt
+	// detect multiple languages. Base.Lang.vtt
+	// TODO: give subtitles descriptive names (lang) and track attributes
+	afs := &afero.Afero{Fs: i.Fs}
+	parentDir := strings.TrimRight(i.Path, i.Name)
+	dir, err := afs.ReadDir(parentDir)
+	if err == nil {
+		base := strings.TrimSuffix(i.Name, ext)
+		r := regexp.MustCompile(base + `\.(.*\.)?vtt`)
```

Comment:
Use `QuoteMeta` to prevent specials characters in the filename being evaluated. Asserting the start and the end of the string prevents invalid matches with filenames contaning a suffix or prefix.

```suggestion
		r := regexp.MustCompile("^" + regexp.QuoteMeta(base) + `\.(.*\.)?vtt$`)
```